vsphere/tests/test_unit.py

```diff
@@ -2814,6 +2816,7 @@ def test_cluster_property_metrics(aggregator, historical_instance, dd_run_check,
     aggregator.assert_metric(
         'vsphere.cluster.configuration.drsConfig.vmotionRate', count=1, value=1, tags=base_tags_cluster_2
     )
+    assert "Could not retrieve properties" not in caplog.text
```

Comment:
What are you asserting here and why at this point?

Reply:
I'm asserting this log line is not called. Before I fixed the code, it was being called because we were trying to collect datacenter property metrics